steering/elections/2022/README.md

```diff
@@ -0,0 +1,83 @@
+# 2022 Istio Steering Committee election guide
+
+The Steering Committee is composed of nine (9) Contribution Seats and four (4) Community Seats. The role of this election is to elect the four Community Seats to the Istio Steering Committee for a one-year term.
+
+## Background
+
+The Istio Steering Committee oversees the administrative aspects of the project, including governance, marketing and community health. Steering exists to allow the Technical Oversight Committee to exclusively focus on the technical aspects of the project.
+
+Some direct responsibilities of steering members to consider as you are deciding whether to run or who to vote for:
+
+- Establishing and enforcing principles to guide the Istio project.
+- Fostering an environment for a healthy and happy community of developers, contributors, vendors, and users.
+- Defining, evolving, and defending a Code of Conduct.
+- Setting marketing and advocacy direction for the project; establishing a publishing schedule and vetting content, encouraging and assisting community members in creating content for conferences, fostering an ecosystem of vendors.
+- Providing neutral mediation as appropriate to try to resolve non-technical disputes that arise as part of the project.
+
+## Schedule
+
+| Date               | Event          |
+|--------------------|----------------|
+| September 19       | Announcement of Election and publication of VOTERS.md |
+| October 2          | Candidate bios and voting exception forms due by 2359 UTC (5pm PDT) |
+| ~1 week            | Election prep week (VOTERS.md validation, Elekto setup and testing) |
+| October 3          | Election begins via Elekto UI |
```

Comment:
Note from October 1 to October 7, Chinese people will be OOO for National Holiday

Reply:
Thanks Zhonghu.  That still leaves almost the whole nomination period, and then allows Oct 7 onwards to vote.  Are you happy to progress with these dates?

Reply:
thanks, SGTM